examples/sol-anchor-contract/README.md

```diff
@@ -0,0 +1,28 @@
+# Pyth SDK Example Program for Solana and the Anchor Library
+
+This example implements the same functionalities as the `sol-contract` example.
+The difference is that this example uses the `anchor` library while the `sol-contract` example uses the lower-level solana interface.
+Please refer to the README of `sol-contract` for a description of the functionalities.
+
+## Run this program
+We assume that you have installed `anchor`, `npm` and `yarn`.
+
+```shell
+# Enter the directory and build this example
+> cd examples/sol-contract-anchor
+> anchor build
+
+# Config the solana CLI by setting its url to devnet
+> solana config set --url https://api.devnet.solana.com
+# Change the `wallet` field in Anchor.toml to your own wallet
+# And then deploy the example contract
+> anchor deploy
+# An error may occur if your wallet does not have enough funds
+
+# Install the client dependencies and invoke this program
+> anchor run install
+> anchor run invoke
+```
+
+You must make sure that the following 3 places have the same program address: (1) Anchor.toml, (2) programs/example-sol-anchor-contract/src/lib.rs and (3) the output of `anchor deploy`.
```

Comment:
I got to the end of the instructions and then run into this.
It's annoying that I now have to modify (1) (2) (3).

Reply:
Let's change the process so that :
Before `anchor build`:
`solana-keygen new -o program_address.json`
Copy paste pubkey from this command, replace it in (1) and (2)
`anchor build`
`anchor deploy --program-keypair program_address.json --program-name example-sol-anchor-contract` this will deploy to the right address